pcs/pcs.go

```diff
@@ -76,16 +76,17 @@ type TdxTcbInfo struct {
 
 // TcbInfo struct is used to map response from tcbInfo field
 type TcbInfo struct {
-	ID                      string     `json:"id"`
-	Version                 byte       `json:"version"`
-	IssueDate               time.Time  `json:"issueDate"`
-	NextUpdate              time.Time  `json:"nextUpdate"`
-	Fmspc                   string     `json:"fmspc"`
-	PceID                   string     `json:"pceId"`
-	TcbType                 byte       `json:"tcbType"`
-	TcbEvaluationDataNumber int        `json:"tcbEvaluationDataNumber"`
-	TdxModule               TdxModule  `json:"tdxModule"`
-	TcbLevels               []TcbLevel `json:"tcbLevels"`
+	ID                      string              `json:"id"`
+	Version                 byte                `json:"version"`
+	IssueDate               time.Time           `json:"issueDate"`
+	NextUpdate              time.Time           `json:"nextUpdate"`
+	Fmspc                   string              `json:"fmspc"`
+	PceID                   string              `json:"pceId"`
+	TcbType                 byte                `json:"tcbType"`
+	TcbEvaluationDataNumber int                 `json:"tcbEvaluationDataNumber"`
+	TdxModule               TdxModule           `json:"tdxModule"`
+	TdxModuleIdentities     []TdxModuleIdentity `json:"tdxModuleIdentities"`
```

Comment:
Added TdxModuleIdentities in TCB Info struct which is required for updated verification logic.